### ISSUE TITLE
01_layers: create docments

### ISSUE THREAD

**Issue by warner-benjamin** [muellerzr/fastai-docment-sprint#5]
Add docments to [01_layers](https://github.com/muellerzr/fastai-docment-sprint/blob/master/nbs/01_layers.ipynb).

See the style guide and contribution guide for more details (links will be added when created).

This requires a moderate to advanced familiarity with both fastai and PyTorch, as the methods cover a wide range of fastai implementations and customizations of PyTorch layers.

Docments progress:
- [ ] Basic manipulations and resize
- [ ] BatchNorm layers
- [ ] Inits
- [ ] Convolutions
- [ ] Embeddings
- [ ] Self attention
- [ ] PixelShuffle
- [ ] Sequential extensions
- [ ] Concat
- [ ] Ready-to-go models
- [ ] Time Distributed Layer
- [ ] Swish and Mish
- [ ] Helper functions for submodules
